ui/jobs/jobs.js

```diff
@@ -173,7 +63,7 @@ let kAnybodyAbilityRegex = null;
 let kMobGainsEffectRegex = null;
 let kMobLosesEffectRegex = null;
 
-let kStatsRegex = Regexes.statChange();
+const kStatsRegex = Regexes.statChange();
```

Comment:
If you want to change declarations to be `const` instead of `let`, I think it would be better to do that in a separate patch rather than included here with these python changes.  Similarly, I don't think it's worth updating the formatting of `jobs.html` just to add an include.

See: https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#pull-requests

Reply:
I have absolutely no attachment to the JS or html changes.  Those are things that eslint did using the config in the repo, so I figured that was how it was supposed to be.

Reply:
That's very surprising to me, as I have never seen eslint change `let` to `const` automatically.  Maybe you have something else running locally that made these changes?  In any case, I think they shouldn't be part of this pull request.  I think if you want to make these changes, it'd be better to make them in a separate pull request.

Reply:
there is a eslint rule called "prefer-const"
```json
{
  "prefer-const": ["error", {
    "destructuring": "any",
    "ignoreReadBeforeAssign": false
  }]
}
```

and you can automatically convert `let` to `const` by `eslint --fix`